SuperblocksHQ/superblocks-lab
# Fix transactions history

### Description of the Change
Fix displaying transaction history. This PR only fixes transactions which come from deploying contracts. Transactions from preview and interact panel still needs to be added.

- Added 2 new actions `ADD_TRANSACTION` and `UPDATE_TRANSACTION` which updates transaction after it receives receipt. Going from pending to successful/failed.
- Transactions are now saved in store, model of transaction can be found in `transaction.model.ts`
- Actions are fired in `deployContract.epicLib.ts`
- Constructor args are not showing up for now, as we don't have configure panel yet




### Verification Process
1. Create several contracts
2. Try to deploy to browser/external network
3. Transactions should be rendered in transaction panel
